projects/go-lib/src/lib/components/go-select/go-select.component.html

```diff
@@ -0,0 +1,18 @@
+<ng-select
+  [items]="items"
+  bindLabel="{{bindLabel}}"
+  [multiple]="multiple"
+  [formControl]="control">
+</ng-select>
+
+<go-hint *ngFor="let hint of hints" [message]="hint" [theme]="theme"></go-hint>
+
+<div *ngIf="control?.errors?.length">
```

Comment:
We probably don't need this div here, could we substitute it for an `ng-container` instead?

Reply:
I copied it from the go-input. I thought we could keep it for consistency